wallet/src/de/schildbach/wallet/ui/BlockListAdapter.java

```diff
@@ -253,21 +277,32 @@ else if (address != null)
 
     public interface OnClickListener {
         void onBlockMenuClick(View view, StoredBlock block);
+
+        void onBlockClicked(BlockInfo blockInfo);
     }
 
     public static class BlockViewHolder extends RecyclerView.ViewHolder {
+
+        /*
         private final ViewGroup transactionsViewGroup;
```

Comment:
@sambarboza ,
do we need this commented out code in the repo yet?

Reply:
@tomasz-ludek, I just kept this code because it will be potentially used, at least as reference, to implement the screen that shows transaction information in blocks.